stdlib/InteractiveUtils/test/runtests.jl

```diff
@@ -331,6 +331,16 @@ if Sys.ARCH === :x86_64 || occursin(ix86, string(Sys.ARCH))
     @test occursin(rgx, output)
 end
 
+using InteractiveUtils: gen_call_with_extracted_types
+
+@testset "gen_call_with_extracted_types" begin
+    for ex0 in ["", 0, 1.0]
+        exret = gen_call_with_extracted_types(Main, :dummy, ex0)
+        @test exret.head == :call
+        @test exret.args[1] == :error
+    end
+end
```

Comment:
Perhaps better to test what caused the failure in the first place instead of this internal function? I think a test like
```julia
@test_throws ErrorException("expression is not a function call or symbol") @less 1
```
might be better?